sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/ingestion/models/Exception.java

```diff
@@ -43,9 +43,9 @@
     private String message;
 
     /**
-     * Raw stack trace. Sent when the frames property is either missing or unreliable.
+     * The path to the minidump file. Used for reports from the NDK.
      */
-    private String stackTrace;
```

Comment:
You cannot rename stacktrace, it is used for legit Java exceptions.
You can use a new field that is not serialized to JSON instead. We were using stacktrace as a temporary field just to hold file path, so if you want a clean solution you need to restore stacktrace as before and use a separate field that is just in memory.